tests/timeseries/_formula_engine/test_formula_composition.py

```diff
@@ -171,6 +171,61 @@ async def test_formula_composition_missing_bat(self, mocker: MockerFixture) -> N
 
         assert count == 10
 
+    async def test_formula_composition_min_max(self, mocker: MockerFixture) -> None:
+        """Test the composition of formulas with min/max values."""
+        mockgrid = MockMicrogrid(grid_side_meter=True)
+        await mockgrid.start(mocker)
+
+        logical_meter = microgrid.logical_meter()
+        engine_min = logical_meter.grid_power.min(Power.from_watts(0)).build(
+            "grid_power_min"
+        )
+        engine_min_rx = engine_min.new_receiver()
+        engine_max = logical_meter.grid_power.max(Power.from_watts(0)).build(
+            "grid_power_max"
+        )
+        engine_max_rx = engine_max.new_receiver()
+
+        await mockgrid.mock_resampler.send_meter_power([100.0])
+
+        # Test min
+        min_pow = await engine_min_rx.receive()
+        assert (
+            min_pow is not None
+            and min_pow.value is not None
+            and math.isclose(min_pow.value.base_value, 0.0)
+        )
+
+        # Test max
+        max_pow = await engine_max_rx.receive()
+        assert (
+            max_pow is not None
+            and max_pow.value is not None
+            and math.isclose(max_pow.value.base_value, 100.0)
```

Comment:
`isclose()` is avalilable for quantities since SDK v0.23.0

Reply:
@daniel-zullo-frequenz but the `isclose()` inside SDK is a private implementation that's not exposed. Not sure how you want to handle such cases for SDK's own test cases though.

Reply:
`isclose()` is a public functionality for `quantities`, see https://github.com/frequenz-floss/frequenz-sdk-python/blob/v0.x.x/src/frequenz/sdk/timeseries/_quantities.py#L131C9-L131C16

So my suggestion was to write that line as `max_pow.value.isclose(Power.from_watts(100.0))`  or something around that

Reply:
>  the isclose() inside SDK is a private implementation that's not exposed.

Where did you get that from? Maybe you got confused by `is_close_to_zero()` which it is indeed a SDK internal function?